README.md

```diff
@@ -1,70 +1,27 @@
-# Getting Started with Create React App
+# React Calculator
+React functional calculator
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![calculator](https://user-images.githubusercontent.com/53324035/107271922-ec3c4980-6a1a-11eb-82b3-402cc5254117.png)
 
-## Available Scripts
+## Built with 
 
-In the project directory, you can run:
+  * Node.js
+  * React
+  * React-DOM
+  * React-Create-App
+  * npm
+  * CSS
+  * ES6
 
-### `yarn start`
+## See a live demo [Here](https://flakster-calculator.herokuapp.com/)
 
```

Comment:
Kindly improve your README file by including the steps to git clone your repo and necessary installation to set up the project locally.

Set it in such a way a novice can run this app on his/her PC.